demos/Demo.ts

```diff
@@ -84,7 +84,11 @@ class Demo {
         document.querySelector('button#sign-transaction-popup').addEventListener('click', async () => {
             const txRequest = generateSignTransactionRequest();
             try {
-                const result = await demo.client.signTransaction(txRequest);
+                const result = await demo.client.signTransaction(
+                    new Promise<SignTransactionRequest>((resolve, reject) => {
+                        window.setTimeout(() => resolve(txRequest), 5000);
```

Comment:
5s is a bit long, how about 2s?

Reply:
It is a demo... I can change it to 2s though.